kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/int-simplification.k

```diff
@@ -169,6 +192,13 @@ module INT-SIMPLIFICATION-COMMON
       requires 0 <=Int A andBool 0 <=Int B andBool 0 <Int C andBool A <=Int D andBool B <=Int C
       [simplification, preserves-definedness]
 
+    rule ( A /Int B ) /Int C => 0
+        requires 0 <=Int A
+         andBool 0 <Int B
+         andBool 0 <Int C
+         andBool A <Int ( C *Int B )
+        [simplification, symbolic(A, B), concrete(C), preserves-definedness]
+
```

Comment:
This is a fairly specific lemma, but I suppose it might be worth having if we run into an expression like this again.